docs/resources/update_resource.md

```diff
@@ -3,14 +3,14 @@ page_title: "azapi_update_resource Resource - terraform-provider-azapi"
 subcategory: ""
 description: |-
   This resource can manage a subset of any existing Azure resource manager resource's properties.
-  -> Note This resource is used to add or modify properties on an existing resource. When azapi_update_resource is deleted, no operation will be performed, and these properties will stay unchanged. If you want to restore the modified properties to some values, you must apply the restored properties before deleting.
+  -> Note This resource is used to add or modify properties on an existing resource. When delete azapi_update_resource, no operation will be performed, and these properties will stay unchanged. If you want to restore the modified properties to some values, you must apply the restored properties before deleting.
 ---
 
 # azapi_update_resource (Resource)
 
 This resource can manage a subset of any existing Azure resource manager resource's properties.
 
--> **Note** This resource is used to add or modify properties on an existing resource. When `azapi_update_resource` is deleted, no operation will be performed, and these properties will stay unchanged. If you want to restore the modified properties to some values, you must apply the restored properties before deleting.
```

Comment:
This doc is changed in https://github.com/Azure/terraform-provider-azapi/pull/841/files, but the source is not changed, so the `make docs` check failed. Would you please help fix it here: https://github.com/Azure/terraform-provider-azapi/blob/main/internal/services/azapi_update_resource.go#L86? Thanks

Reply:
That is strange, I ran make docs on my machine...

Reply:
I will revert the change and re-submit

Reply:
> That is strange, I ran make docs on my machine...

Yes, the `make docs` still works. But the generated docs overwrites the manual changes in https://github.com/Azure/terraform-provider-azapi/pull/841/files